server/src/modules/metadataModule/testRequest1.json

```diff
@@ -0,0 +1,20 @@
+{
+    "paths": [
+        "C:/Users/krzys/Documents/Repos/tk-2022/server/resources/exampleImages/bike.jpg",
+        "C:/Users/krzys/Documents/Repos/tk-2022/server/resources/exampleImages/bus.jpg",
+        "C:/Users/krzys/Documents/Repos/tk-2022/server/resources/exampleImages/feather.jpg",
+        "C:/Users/krzys/Documents/Repos/tk-2022/server/resources/exampleImages/fish.jpg",
+        "C:/Users/krzys/Documents/Repos/tk-2022/server/resources/exampleImages/flower1.jpg",
+        "C:/Users/krzys/Documents/Repos/tk-2022/server/resources/exampleImages/flower2.jpg",
+        "C:/Users/krzys/Documents/Repos/tk-2022/server/resources/exampleImages/lizard1.jpg",
+        "C:/Users/krzys/Documents/Repos/tk-2022/server/resources/exampleImages/lizard2.jpg",
+        "C:/Users/krzys/Documents/Repos/tk-2022/server/resources/exampleImages/lizard3.jpg",
+        "C:/Users/krzys/Documents/Repos/tk-2022/server/resources/exampleImages/ptsd.jpg",
+        "C:/Users/krzys/Documents/Repos/tk-2022/server/resources/exampleImages/trees.jpg"
```

Comment:
Get absolute path from variable because not everyone is "krzys".

Reply:
Also, tests should be in `tests` dir.

Reply:
![obraz](https://user-images.githubusercontent.com/26521377/162613546-06007fe6-b882-4c14-a059-143e2fecb2a0.png)
These files are even used?

Reply:
No, I've added them by accident. Removed.